pkg/ipam/poolallocator/allocator.go

```diff
@@ -394,3 +488,20 @@ func (a *IPPoolAllocator) HasContainer(containerID string) (bool, error) {
 	}
 	return false, nil
 }
+
+// CheckReserved checks whether an IP was reserved with specified owner. It returns error if the resource crd fails to be retrieved.
+func (a *IPPoolAllocator) CheckReserved(reservedOwner v1alpha2.IPAddressOwner) (net.IP, error) {
```

Comment:
I feel `getReservedIP` sounds more proper for what the method actually does.

Reply:
Fixed.